### PR TITLE
added Microsoft Publisher 

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6623

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @mypil, for your information

Thank you

File: windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Microsoft has made a concerted effort to enlighten several of our more popular a
 - Microsoft Remote Desktop
 
 > [!NOTE]
-> Microsoft Visio, Microsoft Office Access and Microsoft Project are not enlightended apps and need to be exempted from WIP policy. If they are allowed, there is a risk of data loss. For example, if a device is workplace-joined and managed and the user leaves the company, metadata files that the apps rely on remain encrypted and the apps stop functioining.
+> Microsoft Visio, Microsoft Office Access, Microsoft Project and Microsoft Publisher are not enlightended apps and need to be exempted from WIP policy. If they are allowed, there is a risk of data loss. For example, if a device is workplace-joined and managed and the user leaves the company, metadata files that the apps rely on remain encrypted and the apps stop functioining.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip.md:77]:
```suggestion
> Microsoft Visio, Microsoft Office Access, Microsoft Project, and Microsoft Publisher are not enlightened apps and need to be exempted from WIP policy. If they are allowed, there is a risk of data loss. For example, if a device is workplace-joined and managed and the user leaves the company, metadata files that the apps rely on remain encrypted and the apps stop functioning.
```

**Comment by mypil** [comment]
@Dansimp - PR is ready for final review. This is based on issue #6623.

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@Dansimp - Can you please merge this approved PR?

Thank you.

**Pull request merged**